src/scribe_data/cli.py

```diff
@@ -19,3 +19,178 @@
     * along with this program.  If not, see <https://www.gnu.org/licenses/>.
     -->
 """
+
+#!/usr/bin/env python3
+
+import sys
+import os
+import argparse
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.dirname(current_dir)
+sys.path.append(parent_dir)
+
+DATA_DIR = Path('language_data_export')
+
+# Mapping of possible inputs to standardized language names
+LANGUAGE_MAP = {
+    'en': 'English', 'english': 'English',
+    'fr': 'French', 'french': 'French',
+    'de': 'German', 'german': 'German',
+    'it': 'Italian', 'italian': 'Italian',
+    'pt': 'Portuguese', 'portuguese': 'Portuguese',
+    'ru': 'Russian', 'russian': 'Russian',
+    'es': 'Spanish', 'spanish': 'Spanish',
+    'sv': 'Swedish', 'swedish': 'Swedish'
+}
+
+def list_languages() -> None:
+    if not DATA_DIR.exists() or not DATA_DIR.is_dir():
+        print(f"Directory '{DATA_DIR}' does not exist.")
+        return
+
+    languages = [lang.name for lang in DATA_DIR.iterdir() if lang.is_dir()]
+    languages.sort()
+    print("Available languages:")
+    for lang in languages:
+        print(f"- {lang}")
+
+def list_word_types(language: str = None) -> None:
+    if language:
+        # Normalize the input language
+        normalized_language = LANGUAGE_MAP.get(language.lower())
+        if not normalized_language:
+            print(f"Language '{language}' is not recognized.")
+            return
+
+        language_dir = DATA_DIR / normalized_language
+        if not language_dir.exists() or not language_dir.is_dir():
+            print(f"No data found for language '{normalized_language}'.")
+            return
+
+        word_types = [wt.stem for wt in language_dir.glob('*.json')]
+        if not word_types:
+            print(f"No word types available for language '{normalized_language}'.")
+            return
```

Comment:
@wkyoshida, @andrewtavis, shall I update the `language_meta_data.json` then?

Reply:
Note that I moved away from this in the most recent commits as I realized that we were gonna have an issue where if we didn't update the `language_metadata.json` (note the rename) file, then we'd lose functionality... Let's check on this, but I think a great way to go about this would be to assure that `language_data_extraction` has a directory for each word type, and then from there we can just read in the structure for the language-word type combinations 😊